src/main/java/org/prebid/server/spring/config/PrometheusMapperConfiguration.java

```diff
@@ -0,0 +1,297 @@
+package org.prebid.server.spring.config;
+
+import io.prometheus.client.dropwizard.samplebuilder.CustomMappingSampleBuilder;
+import io.prometheus.client.dropwizard.samplebuilder.DefaultSampleBuilder;
+import io.prometheus.client.dropwizard.samplebuilder.MapperConfig;
+import io.prometheus.client.dropwizard.samplebuilder.SampleBuilder;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Configuration
+public class PrometheusMapperConfiguration {
+
+    @Bean
+    @ConditionalOnProperty(prefix = "metrics.prometheus.labels", name = "enabled",
+            havingValue = "false", matchIfMissing = true)
+    public SampleBuilder defaultSampleBuilder() {
+        return new DefaultSampleBuilder();
+    }
+
+    @Bean
+    @ConditionalOnProperty(prefix = "metrics.prometheus.labels", name = "enabled", havingValue = "true")
+    public SampleBuilder labelsSampleBuilder() {
+        final List<MapperConfig> mapperConfigs = new ArrayList<>();
+
+        mapperConfigs.addAll(accountModuleMapperConfigs());
+        mapperConfigs.addAll(moduleMapperConfigs());
+        mapperConfigs.addAll(accountMapperConfigs());
+        mapperConfigs.addAll(adapterMapperConfigs());
+        // other
+        mapperConfigs.add(requestsOk);
+        mapperConfigs.add(bidderCardinality);
+        mapperConfigs.addAll(privacyMappingConfigs());
+        mapperConfigs.addAll(vertxResponseConfig);
+        mapperConfigs.addAll(vertxRequestConfigs);
+        mapperConfigs.add(vertxServer);
+
+        return new CustomMappingSampleBuilder(mapperConfigs);
+    }
```

Comment:
Done 👍🏻 But I had to use a `List<List<MapperConfig>>` because Spring didn't like the max of
```
@Bean
public MapperConfig mapperConfig() {}
```
and
```
@Bean
public List<MapperConfig> mapperConfig() {}
```
so that the last Bean overwrites all other Bean declarations